src/plugins/intel_cpu/src/nodes/executors/interpolate.cpp

```diff
@@ -111,9 +111,9 @@ float ov::intel_cpu::InterpolateExecutor::coordTransToInput(int outCoord,
     case InterpolateCoordTransMode::pytorch_half_pixel: {
         if (outShape > 1) {
             return (outCoord + 0.5f) / scale - 0.5f;
-        } else {
-            return 0;
         }
+        return 0;
+
```

Comment:
I guess we do not need breaks in this switch case at all

Reply:
Removed breaks after return